test/cloud.jl

```diff
@@ -9,5 +9,5 @@ Gurobi.free_env(env)
 
 @test_throws ErrorException Gurobi.Env("fakehostdoesnotexist", "wrongpassword")
 
-solver = GurobiSolver(cloudhost="csdemo.gurobi.com")
-LinearQuadraticModel(solver)
+include(joinpath(Pkg.dir("MathProgBase"),"test","linproginterface.jl"))
+linprogsolvertest(GurobiSolver(cloudhost="csdemo.gurobi.com"))
```

Comment:
Yup! :smile: (subject to the `LD_LIBRARY_PATH` thing)

At least most of the time... I ran into some issues initially with server capacity, but they seem to have disappeared after I removed the `LinearQuadraticModel` lines. And the issues with server capacity seemed to be caused by the test, as opposed to other users: the server reported that I was first in the queue, I would not get to the front for more than a few minutes, but once I force-stopped the test, the server seemed to have capacity again.

But I can't reproduce this issue now, it's passed the last few times I've run it.